### ISSUE TITLE
Duplicate/migrating/broken table tools

### ISSUE THREAD

**Issue by blackraine** [ThePants999/tts-comp-ftc-base-lua-scripts#1]
In a recent game, several of the table tools (faction card deck selector, mission selector, score sheet) were duplicated and the extra score sheet apparently had physics turned on and was colliding with other tool objects.

Additionally, the phase advance button forgot what phase it was and stopped working for one player while the other player reported it changing location each time they clicked on it before it disappeared completely.

Finally, the command point tracker and victory point tracker displayed command points only as 8, 18, 28, etc and the tooltip text displayed different command point totals depending on where you hovered (so, likely also duplicated as anything but 0, 1, or 7 overlaid on a 0 looks like an 8).

**Comment by ThePants999** [comment]
Are you sure you were using this mod instead of the original FTC base? This mod no longer has a faction card deck selector.

**Comment by blackraine** [comment]
![20220202100808_1](https://user-images.githubusercontent.com/30580049/152192271-e97fb1a0-b20b-478d-a5ca-afbcc95ed68b.jpg)

So, I loaded up my saved game to show what's happening with it.  However, I also loaded up a fresh game and the fresh game was fine. I don't know why or how Tabletop Simulator would be caching items from the old FTC and loading them with the new FTC, but I had torn down and restarted the game multiple times to try and get it to load right.
